src/lib/y2users/linux/writer.rb

```diff
@@ -158,6 +165,68 @@ def add_user(user, issues)
         log.error("Error creating user '#{user.name}' - #{e.message}")
       end
 
+      # Command for creating new users
+      GROUPADD = "/usr/sbin/useradd".freeze
+      private_constant :USERADD
+
+      # Executes the command for creating the group
+      #
+      # @param group [Y2User::Group] the group to be created on the system
+      # @param issues [Y2Issues::List] a collection for adding an issue if something goes wrong
+      def add_group(group, issues)
```

Comment:
Thinking loud: Maybe the logic for creating, editing and removing users/groups deserves its own classes, something like `Action::Create`, `Action::Edit`, `Action::Delete`. The writer is growing and adding a lot logic.

Reply:
if it grow too much, then it is fine for me. But still it is writer specific so it probably needs to be under linux namespace and then `Y2Users::Linux::Action::Create` looks quite long for me.

Reply:
Inside the writer it would be `Action::Create` only, but it was only a wild idea, nothing to be done now.

Reply:
I expect we will introduce it when we add also removal of user